CHANGELOG.rst

```diff
@@ -7,10 +7,13 @@ dev
 New features
 
 * Improved the functionality of `merge_double_ended`, by adding a check that handles measurements missing in one channel while present in the other ([#171](https://github.com/dtscalibration/python-dts-calibration/pull/171))
+* Support for Python 3.11
+* Requiring netcdf4 >= 1.6.4
```

Comment:
Not really a new feature I'd think?

I think I like such a layout more for changelogs:

**Fixed:**
 - Bug in abc

**Added:**
 - Support for Python 3.11

**Changed:**
 - Now requires `netcdf >= 1.6.4`